trigger/patch.go

```diff
@@ -190,10 +190,16 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 	if api.ProjectIdentifier != nil {
 		projectName = utility.FromStringPtr(api.ProjectIdentifier)
 	}
-	collectiveStatus, err := patch.CollectiveStatus(t.patch.Id.Hex())
-	if err != nil {
-		return nil, errors.Wrap(err, "getting collective status for patch")
+
+	collectiveStatus := t.patch.Status
```

Comment:
Should this maybe be t.data.status? That's what it was before we switched it to collective status here.

Reply:
i think data.status is from the subscription so it would be more accurate to have patch.status maybe? open to discussion

Reply:
we should probably use t.data.status to remain consistent with other makeData functions

Reply:
updated!